### ISSUE TITLE
Running rocker/shiny as shiny user (non root)

### ISSUE THREAD

**Issue by OceaneCsn** [rocker-org/rocker-versioned2#360]
Hello, 

I am trying to run a rocker/shiny:4.0.2 container as a shiny user instead of root.

When running `sudo docker run --user shiny --rm -p 3838:3838 rocker/shiny:4.0.2`, I have the following error : 
`s6-mkdir: warning: unable to mkdir /var/run/s6: Permission denied`.

I noticed it was already reported in [this issue](https://github.com/rocker-org/shiny/issues/79), so I tried :  
`sudo docker run --user shiny -p 3838:3838 --rm rocker/shiny /usr/bin/shiny-server`.

The server starts, however I can see that the process is not owned by shiny, but by systemd-c instead.

Like in [this Dockerfile](https://github.com/randy3k/shiny-cloudrun-demo/blob/master/Dockerfile), I built a new container using a custom shiny server config file, and specifying USER shiny : 

```
FROM rocker/shiny:4.0.2
COPY shiny-customized.config /etc/shiny-server/shiny-server.conf
USER shiny
CMD ["/usr/bin/shiny-server"]
```

shiny-customized.config:
```
run_as shiny;
server {
  listen 3838;

  location / {
    site_dir /srv/shiny-server;
    log_dir /var/log/shiny-server;
    directory_index off;
  }
}
```
Unfortunately, the process launched by `sudo docker run --user shiny --rm -p 3838:3838 shiny_container` is still owned by systemd-c.
My problem is that I have a shiny app I would like to deploy using rocker/shiny, but my app is supposed to write its logs in a folder owned by shiny, so I end up with `permission denied` errors. Any idea on how to get shiny-server running as a shiny user process?

My OS details : 
```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal
```
This might by OS related, as it works on this distribution :  

```
Distributor ID: Debian
Description:    Debian GNU/Linux 9.13 (stretch)
Release:        9.13
Codename:       stretch
```

**Comment by eitsupi** [comment]
Duplicate of #81. Fixed by #247.

**Comment by eitsupi** [comment]
Thank you for reporting this. I'm glad it worked well.
